data/projects.js

```diff
@@ -419,5 +419,26 @@ coala should support generating metrics for your code.",
 		"tags" : ["Jetbrains IDEs", "Plugins"],
 		"markdown": "jetbrains-ide-plugin.md",
 		"collaborating_projects" : ["coala", "jetbrains"]
+	},
+	{
+		"name" : "One click linting for online projects in the browser",
+		"desc" : "Create a loader for projects from remote data sources \
+		that can be used from the coala web page in order to provide one click linting \
+		for online projects in collaboration with the Roberta outreach project for young \
```

Comment:
I'm not sure if the description will fit the cards. Maybe remove the "in collaboration..." part. Roberta is in the collab projects entry and mentioned in the md file.